main.go

```diff
@@ -273,14 +273,14 @@ func Main(ctx context.Context, reader InputReader) error {
 	ctx = initOtelContext(ctx)
 
 	// add additional attributes if provided to the runtime attributes
-	if addAttributes != "" {
-		addAttrs := strings.Split(addAttributes, ",")
+	if additionalAttributes != "" {
+		addAttrs := strings.Split(additionalAttributes, ",")
 		for _, attr := range addAttrs {
 			kv := strings.Split(attr, "=")
 			if len(kv) == 2 {
 				runtimeAttributes = append(runtimeAttributes, attribute.Key(kv[0]).String(kv[1]))
 			} else {
-				log.Fatalf("FATAL: invalid attribute provided: %s", attr)
+				return fmt.Errorf("invalid attribute provided: %s", attr)
```

Comment:
suggestion: you could use an slice of errors and append the new errors here, so the consumer will have all the wrong attributes in a row.

Reply:
Done